docs/user-documentation/soc-and-feeds.md

```diff
@@ -0,0 +1,126 @@
+id: soc-and-feeds
+slug: /soc-and-feeds
+sidebar_label: SOC and Feeds
+---
+
+import Tabs from '@theme/Tabs'
+import TabItem from '@theme/TabItem'
+
+Swarm provides the ability to store content in content-addressed chunks or single owner chunks (SOC). With single owner chunks, a user can assign arbitrary data to an address and attest chunk integrity with their digital signature.
+
+Feeds are a unique feature of Swarm. They constitute the primary use case for single owner chunks. Feeds can be used for versioning revisions of a mutable resource, indexing sequential updates to a topic, publish the parts to streams, or post consecutive messages in a communication channel to name but a few. Feeds implement persisted pull-messaging and can also be interpreted as a pub-sub system.
+
+Because Feeds are built on top of SOCs, their interfaces in Bee is somewhat similar and they are using common concepts.
+
+## Single Owner Chunks
+
+The address of an SOC is calculated as the hash of an `identifier` and and `owner`. The `identifier` is a 32 bytes arbitrary data, usually expected as a hex string or in a `Uint8Array` type. The `owner` is the Ethereum address of the owner of the chunk, which is 20 bytes expected as a hex string or in a `Uint8Array` type.
+
+### Reading SOCs
+
+In order to read data from an SOC we need to make a reader object for a certain `owner`, then we can download the data with the provided `identity` that identifies the chunk.
+
+```js
+const owner = '0x8d3766440f0d7b949a5e32995d09619a7f86e632'
+const socReader = bee.makeSOCReader(owner)
+const identity = '0000000000000000000000000000000000000000000000000000000000000000'
+const soc = await socReader.download(identity)
+const data = soc.payload()
+```
+
+### Writing SOCs
+
+When writing an SOC first we need to make a writer object. Because we need to sign the chunk we need to pass in a `signer` object. The `signer` object can be either an Ethereum private key (as a hex string or `Uint8Array` type) or it can be an instance of the `Signer` interface. The `Signer` interface can be used for integration with 3rd party Ethereum wallet applications, because Swarm uses the same format for signing chunks that Ethereum uses for signing transactions.
+
+```ts
+type SyncSigner = (digest: Uint8Array) => Signature
+type AsyncSigner = (digest: Uint8Array) => Promise<Signature>
+
+/**
+ * Interface for implementing Ethereum compatible signing.
+ *
+ * @property sign     The sign function that can be sync or async
+ * @property address  The ethereum address of the signer
+ */
+export type Signer = {
+  sign: SyncSigner | AsyncSigner
+  address: EthAddress
+}
+```
+
+:::warning Your communication privacy may be at risk
+We suggest to use either ephemeral private keys (e.g. randomly generated) or the `Signer` interface when writing to SOC or Feeds. Never use your real Ethereum private keys here (or in any web applications really) directly because you may lose all your Ethers belonging to it.
```

Comment:
```suggestion
We suggest to use either ephemeral private keys (e.g. randomly generated) or the `Signer` interface when writing to SOC or Feeds. Never use your real Ethereum private keys here (or in any web applications really) directly because you may lose your funds stored on it.
```